src/frontends/ir/CMakeLists.txt

```diff
@@ -7,3 +7,8 @@ ov_add_frontend(NAME ir
                 LINK_LIBRARIES openvino::pugixml
                                # TODO: remove dependency below in CVS-69781
                                openvino::runtime::dev)
+
+
+if(ENABLE_TESTS)
+    add_subdirectory(tests)
```

Comment:
I suppose if we go this way, the cmake `ov_add_frontend ` function above will also grad tests files and includes it into frontend.

Reply:
I suppose this should help https://github.com/openvinotoolkit/openvino/pull/13198